packages/web-components/src/components/locale-modal/region-item.ts

```diff
@@ -60,8 +73,14 @@ class C4DRegionItem extends C4DCard {
     return disabled
       ? this._renderDisabledLink()
       : html`
-          <button id="link" class="${classes}" type="button">
-            <div class="${cardClasses}">${this._renderInner()}</div>
+          <button
+            id="link"
+            class="${classes}"
+            type="button"
+            part="button enabled-button">
```

Comment:
Lets follow a convention we've been using elsewhere:
```suggestion
            part="button button--enabled">
```